.github/scripts/branch_pr_issue_closer.py

```diff
@@ -447,7 +470,20 @@ def _main_prog():
             print("Issue #{} has been closed.".format(issue_num))
         else:
             #Extract card id from id dictionary:
-            card_id = proj_issue_card_ids[issue_num]
+            try:
+                card_id = proj_issue_card_ids[issue_num]
+            except KeyError:
+                #If there is a key error, then it means the issue
```

Comment:
Why use `try` / `except` here instead of a test? E.g., 
```suggestion
            if issue_num in proj_issue_card_ids:
                card_id = proj_issue_card_ids[issue_num]
            else:
                #If there is a key error, then it means the issue
```

Reply:
Good call.  Fixed.